### PR TITLE
Remove common clause.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

**Review by czpmango — DISMISSED** [review]

File: .clang-format [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Copyright (c) 2020 vesoft inc. All rights reserved.
 #
 # This source code is licensed under Apache 2.0 License,
```
Comment (yixinglu) [inline .clang-format]: replace `,` with `.`

Comment (Shylock-Hg) [reply .clang-format]: Fixed.

File: .licenserc.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - '**/*.md'
+    - 'third-party'
+    - 'LICENSE'
+    - 'NOTICE'
+    - 'certs'
+    - 'src/interface/gen-cpp2'
+    - '.gitkeep'
+    - 'include/common/datatypes/ValueOps-inl.h'
+
+  comment: on-failure
```
Comment (Aiee) [inline .licenserc.yaml]: Add a newline maybe?

**Review by Aiee — DISMISSED** [review]

File: .licenserc.yaml (hunk 2 above)
Comment (Shylock-Hg) [reply .licenserc.yaml]: Fixed.

**Review by Aiee — APPROVED** [review]

**Pull request merged**